netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -337,7 +337,7 @@ _net_ebpf_extension_sock_addr_program_info_on_client_detach(
     _In_ const net_ebpf_extension_program_info_client_t* detaching_client)
 {
     UNREFERENCED_PARAMETER(detaching_client);
-    _net_ebpf_sock_addr_get_program_context = NULL;
+    // _net_ebpf_sock_addr_get_program_context = NULL;
```

Comment:
Should this be commented, if the intent is to keep the line as a history/reference?

Reply:
Please remove all commented out code as per coding guide.

Reply:
Yes this was unintended. Fixed.